plugins/legacy-plugin-chart-time-table/types/external.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.png';
+declare module '@data-ui/sparkline';
+declare module 'reactable-arc'
+declare module 'mustache';
```

Comment:
Nit: missing new line (Prettier didn't catch this because `*.d.ts` is in `.prettierignore` -- maybe we should remove it).